### PR TITLE
add createComment feature

### PR DESCRIPTION
**Pull request by a36168120**
add createComment feature

### CODE REVIEW

File: client/js/components/CommentList/CommentList.js [hunk 1]
```diff
@@ -5,34 +5,32 @@ import styles from './styles';
...
+            <View style={{width: 2}} />
+            <Text>{comment.likes.length} likes</Text>
           </View>
+
+          <TouchableOpacity opacity={0.8}>
+            <Text>Like</Text>
+          </TouchableOpacity>
         </View>
       </View>
     </View>
```
Comment (siddharthparmar7) [inline client/js/components/CommentList/CommentList.js:36]: move your styles to styles.js file.

File: client/js/components/CreateComment/CreateComment.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+              value={text}
+              style={styles.input}
+              placeholder={'Share Your Thoughts...'}
+              keyboardType={'default'}
+            />
+            <TouchableOpacity
+              onPress={() => {
+                createComment({
+                  variables: {
+                    authorId: 'ck07255yn0pyx01415da8y14o',
```
Comment (siddharthparmar7) [inline client/js/components/CreateComment/CreateComment.js]: change authorId to dynamic id.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the requested changes.

File: client/js/components/CommentList/CommentList.js (hunk 1 above)
Comment (a36168120) [reply client/js/components/CommentList/CommentList.js:36]: did you mean the  "<View style={{width: 2}} />"

Comment (siddharthparmar7) [reply client/js/components/CommentList/CommentList.js:36]: yes

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**